<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="BoardPlot" name="mainPlot" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>671</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="SidePlot" name="top" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="SidePlot" name="right" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>570</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="SidePlot" name="left" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="SidePlot" name="bottom" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>210</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용자명</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="linkButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>80</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>LINK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>160</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>AUTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manualButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>240</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>MANUAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spotButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>320</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SPOT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>560</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="histogramButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>480</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>HISTOGRAM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>640</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>REPLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyzeButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>400</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ANALYZE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1455</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoardPlot</class>
   <extends>QWidget</extends>
   <header>boardplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SidePlot</class>
   <extends>QWidget</extends>
   <header>sideplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
